packages/tokenMovementMonitor/monitor.go

```diff
@@ -0,0 +1,127 @@
+package tokenMovementMonitor
```

Comment:
By convention, packages are given lower case, single-word names; there should be no need for underscores or mixedCaps. [source](https://golang.org/doc/effective_go.html#package-names)